source/_components/light.niko_home_control.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: "Niko Home Control"
+description: "Instructions on how to integrate Niko Home Control into Home Assistant."
+date: 2018-04-08 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.36
```

Comment:
Incorrect release number, `0.36` is pretty old. The suggestion is to update it to `0.68`.